packages/react-focus/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@fluentui/react-focus",
```

Comment:
Will beachball support publishing to two scopes @kenotron ?

Reply:
i don't see this being a problem.

Reply:
Before this merges, @kenotron needs to make sure that the account we use for publishing has permission to publish to `@fluentui` as well.

Reply:
I need the owner of uifabricteam on npm js.org to accept my invitation. Likely @dzearing?